force-app/main/default/classes/ESA_cls_constants.cls

```diff
@@ -0,0 +1,13 @@
+/**********************************************************************************************
+* @Author:      Deepak 
+* @Date:        23/02/2023
+* @Description: The purpose of this class is to have constant values to be reffered by main class
+* @Revision(s): [Date] - [Change Reference] - [Changed By] - [Description]   
+***********************************************************************************************/
+
+public class ESA_cls_constants {
+
+    public static final String Case_Approval_Time_Unit_Week = 'Week';
```

Comment:
Please follow the below from salesforce:
https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/langCon_apex_constants.htm